server/services/store/sqlstore/initialize.go

```diff
@@ -28,60 +49,88 @@ func (s *SQLStore) InitializeTemplates() error {
 // ToDo: needs a rewrite to loop over teams and insert boards and
 // blocks, commented to move forward with the rest of the store
 // changes
-func (s *SQLStore) importInitialTemplates() error {
-	/*
-		s.logger.Debug("importInitialTemplates")
-		blocksJSON := initializations.MustAsset("templates.json")
-
-		var archive model.Archive
-		err := json.Unmarshal(blocksJSON, &archive)
-		if err != nil {
-			return err
-		}
+func (s *SQLStore) importInitialTemplates(db sq.BaseRunner, teamIDs []string) error {
+	s.logger.Debug("importInitialTemplates")
+	blocksJSON := initializations.MustAsset("templates.json")
 
-		// ToDo: loop over teams
+	var archive model.Archive
+	err := json.Unmarshal(blocksJSON, &archive)
+	if err != nil {
+		return err
+	}
 
-		globalContainer := store.Container{
-			TeamID: "0",
-		}
+	for _, teamID := range teamIDs {
+		for _, board := range archive.Boards {
+			boardBlocks := []model.Block{}
+			newBoardID := utils.NewID(utils.IDTypeBoard)
```

Comment:
As we're generating new board IDs, we should store the ones from the archive file in a `templateID` field for telemetry tracking purposes